index.js

```diff
@@ -28,9 +28,10 @@ var adminDVPermission = require('permission')(['admin', 'data-viewer']);
 
 var app = express();
 
+var disableAuth = process.env.DISABLE_AUTH == 1 || process.env.DISABLE_AUTH == 'true';
 var noAuth = (req, res, next) => next();
 var auth = (req, res, next) => {
-  if (req.user && req.user.username) {
+  if ((req.user && req.user.username) || disableAuth) {
```

Comment:
This should be sufficient to disable auth checks, which should simplify https://github.com/hotosm/OpenMapKitServer/pull/41/files#diff-1685f84cfc659c8bd04ccbd4610c41fcR55

Reply:
I tried to simplify, but as passport doesn't have permission options, and the permissions are managed by another package, it didn't work